packages/api-fetch/README.md

```diff
@@ -0,0 +1,64 @@
+# @wordpress/api-fetch
+
+Wrapper around `window.fetch` to call WordPress REST APIs.
```

Comment:
Minor: Should we maybe be more explicit for general use that this _requires_ `Fetch` (and thus, a polyfill maybe required for older browsers?)

Reply:
yes, definitely, this makes me think we may need a consistent way to gather the required polyfills for each package. Better automated but It's probably not easy to automate.

Reply:
Related: #7159